.code-samples.meilisearch.yaml

```diff
@@ -255,7 +217,44 @@ geosearch_guide_sort_usage_2: |-
   };
 
   SearchResult<Restaurant> restaurants = await client.Index("restaurants").SearchAsync<Restaurant>("restaurants", filters);
-  foreach (var restaurant in restaurants.Hits)
+authorization_header_1: |-
+  MeilisearchClient client = new MeilisearchClient("http://127.0.0.1:7700", "masterKey");
+  var keys = await client.GetKeysAsync();
+get_one_key_1: |-
+  Key key = await client.GetKeyAsync("d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4");
+get_all_keys_1: |-
+  Result keyResult = await client.GetKeysAsync();
+  var keys = keyResult.Result;
+create_a_key_1: |-
+  Key keyOptions = new Key
   {
-      Console.WriteLine(restaurant.Name);
-  }
+      Description = "Add documents: Products API key",
+      Actions = new string[] { "documents.add" },
+      Indexes = new string[] { "products" },
+      ExpiresAt = "2042-04-02T00:42:42Z"
+  };
+  Key createdKey = await this.client.CreateKeyAsync(keyOptions);
+update_a_key_1: |-
```

Comment:
For those who haven't been implemented yet, I had asked the team doc and It said [here](https://github.com/meilisearch/documentation/pull/1181#issuecomment-946792717) that it was better to put some sentence to explain why like `Not implemented yet` or other. That way the user doesn't think the website is broken.
Maybe you can recheck it

Reply:
If the reason is "That way the user doesn't think the website is broken", the `update_a_key_1` is not the only problem since most of the code samples are not implemented for the dotnet!
However I will add it for this one at least :)

Reply:
As you want!